integration/tests/conftest.py

```diff
@@ -548,3 +548,56 @@ def expected_error_checker(accounts, web3_client):
         "common/ExpectedErrorsChecker", "0.8.12", accounts[0], contract_name="A"
     )
     yield contract
+
+
+@pytest.fixture(scope="class")
+def query_account_caller_contract(
+    web3_client: Web3Client,
+    accounts: EthAccounts,
+) -> Contract:
+    contract, _ = web3_client.deploy_and_get_contract(
+        "precompiled/QueryAccountCaller.sol",
+        "0.8.10",
+        contract_name="QueryAccountCaller",
+        account=accounts[0],
+    )
+    return contract
+
+
+@pytest.fixture(scope="class")
+def multiple_actions_erc721(web3_client, accounts):
+    contract, contract_deploy_tx = web3_client.deploy_and_get_contract(
+        "EIPs/ERC721/MultipleActions", "0.8.10", accounts[0], contract_name="MultipleActionsERC721"
+    )
+    return accounts[0], contract
+
+
+@pytest.fixture(scope="session")
+def get_counter_value():
+    def gen_increment_counter():
+        count = 0
+        while True:
+            count += 1
+            yield count
+
+    return gen_increment_counter()
+
+
+@pytest.fixture(scope="class")
+def call_solana_caller(accounts, web3_client):
+    contract, _ = web3_client.deploy_and_get_contract("precompiled/CallSolanaCaller.sol", "0.8.10", accounts[0])
+    return contract
+
+
+def create_resource(contract, salt, sender, owner, web3_client):
```

Comment:
conftest is the place for fixtures

it is better to move create_resource to some helper and rename if it is needed to make the code readable

Reply:
Refactored ✅